### PR TITLE
Add interview information

### PR DESCRIPTION
**Pull request by siddhu001**

### CODE REVIEW

File: _pages/interview.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: page
```
Comment (ftshijt) [inline _pages/interview.md:2]: I'm not sure, but how about make the interview page merged with the position page (or another option is to put it into Info section via `posts`? For the following reasons:
1. this infomration is highly correlated to that one
2. we already have many navigation pages

File: _pages/interview.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+permalink: /interview/
+title: Interview
+nav: true
+order: 8
```
[the remark this answers is not in the document]
Comment (siddhu001) [reply _pages/interview.md]: I removed this page!

File: _pages/interview.md (hunk 1 above)
Comment (siddhu001) [reply _pages/interview.md]: I merged it with the positions page.

**Comment by ftshijt** [comment]
Many thanks! looks cool to me

**Pull request merged**